CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+### 1.1.5 - [#15](https://github.com/openfisca/extension-template/pull/15)
+
+* Add continuous deployment with CircleCI on merge on `master` branch
```

Comment:
> Add continuous deployment with CircleCI on merge on `master` branch

Add continuous deployment with CircleCI,  **triggered by a** merge **to** `master` branch